<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelCreater</class>
 <widget class="QDialog" name="ModelCreater">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>460</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>ModelCreater</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QScrollArea {
	border:none;
	background:transparent;
}

QScrollBar:vertical {
	background:transparent;
	padding:0px;
	width:10px;
}

QScrollBar::handle:vertical{
	background:rgba(0,0,0,35);
	border-radius:5px;
	min-height:30px;
}

QScrollBar::handle:vertical:hover,QScrollBar::handle:vertical:pressed {
	background:rgba(0,0,0,75);
}

QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical {
	background:none;
}

QScrollBar::add-line:vertical,QScrollBar::sub-line:vertical {
	background:none;
}

QRadioButton
{
    spacing: 5px;
	font-size:15px;
}

QRadioButton::indicator
{
    width: 16px;
    height: 16px;
}

QRadioButton::indicator:unchecked
{
	image: url(:/Animism/Resource/checkbox-unchecked.png);
}

QRadioButton::indicator:checked
{
	image: url(:/Animism/Resource/checkbox-indeterminate.png);
}

QCheckBox
{
    spacing: 5px;
	font-size:15px;
}

QCheckBox::indicator
{
    width: 16px;
    height: 16px;
}

QCheckBox::indicator:unchecked
{
	image: url(:/Animism/Resource/checkbox-unchecked.png);
}

QCheckBox::indicator:checked
{
	image: url(:/Animism/Resource/checkbox-checked.png);
}

QCheckBox::indicator:indeterminate
{
	image: url(:/Animism/Resource/checkbox-indeterminate.png);
}

QSpinBox
{
	font-size: 13px;
	border: 1px solid rgb(160,160,160);
	background-color: transparent;
	padding-left: 5px;
	padding-right: 5px;
}

QTextEdit
{
	font-size: 12px;
	border: 1px solid rgb(160,160,160);
	background-color: transparent;
}

QListWidget
{
	font-size: 12px;
	border: 1px solid rgb(160,160,160);
	background-color: transparent;
}

QLineEdit
{
	font-size: 12px;
	border: 1px solid rgb(160,160,160);
	background-color: transparent;
}

QTreeWidget
{
	font-size: 12px;
	border: 1px solid rgb(160,160,160);
	background-color: transparent;
}

QTreeWidget::item 
{
    spacing: 5px;
	font-size:15px;
}

QTreeWidget::indicator
{
    width: 16px;
    height: 16px;
}

QTreeWidget::indicator:unchecked
{
	image: url(:/AnimismAI/Resource/checkbox-unchecked.png);
}

QTreeWidget::indicator:checked
{
	image: url(:/AnimismAI/Resource/checkbox-checked.png);
}

QTreeWidget::indicator:indeterminate
{
	image: url(:/AnimismAI/Resource/checkbox-indeterminate.png);
}

QPushButton
{
	font-size:12px;
}

#scrollArea_event
{
	background-color: transparent;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget_frame" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_app_head" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>7</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_app_icon" native="true">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>727</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="WinButton" name="pushButton_app_close">
           <property name="minimumSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="toolTip">
            <string>关闭</string>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_container" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,1,0">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="horizontalSpacing">
            <number>10</number>
           </property>
           <property name="verticalSpacing">
            <number>0</number>
           </property>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label_proc_tip">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Microsoft YaHei</family>
               <pointsize>-1</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel{font-size:13px;}</string>
             </property>
             <property name="text">
              <string>目标进程</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="lineEdit_proc_info">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>35</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Microsoft YaHei</family>
               <pointsize>-1</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">#lineEdit_proc_info
{
	padding-left: 5px;
	padding-right: 5px;
}</string>
             </property>
             <property name="frame">
              <bool>false</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>选择目标进程</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="IconButton" name="pushButton_proc_open">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>35</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Microsoft YaHei</family>
               <pointsize>-1</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>打开进程列表</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_5">
           <property name="horizontalSpacing">
            <number>10</number>
           </property>
           <property name="verticalSpacing">
            <number>0</number>
           </property>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label_model_path_tip">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Microsoft YaHei</family>
               <pointsize>-1</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel{font-size:13px;}</string>
             </property>
             <property name="text">
              <string>保存位置</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="lineEdit_model_path">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>35</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Microsoft YaHei</family>
               <pointsize>-1</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">#lineEdit_model_path
{
	padding-left: 5px;
	padding-right: 5px;
}</string>
             </property>
             <property name="frame">
              <bool>false</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>选择保存位置</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="IconButton" name="pushButton_model_path_open">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>35</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Microsoft YaHei</family>
               <pointsize>-1</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>浏览</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,1,0" columnstretch="1,0,0">
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="2" column="1">
            <widget class="QCheckBox" name="checkBox_event_ctrl">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>110</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Microsoft YaHei</family>
               <pointsize>-1</pointsize>
              </font>
             </property>
             <property name="text">
              <string>全选/全不选</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <property name="tristate">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>248</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0" colspan="3">
            <widget class="QScrollArea" name="scrollArea_event">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="cursor" stdset="0">
              <cursorShape>ArrowCursor</cursorShape>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_eventmap">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>348</width>
                <height>123</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">#scrollAreaWidgetContents_eventmap
{
	background: transparent;
}</string>
              </property>
             </widget>
            </widget>
           </item>
           <item row="0" column="0" colspan="3">
            <widget class="QLabel" name="label_event_tip">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Microsoft YaHei</family>
               <pointsize>-1</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel{font-size:13px;}</string>
             </property>
             <property name="text">
              <string>事件模型</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>178</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="IconButton" name="pushButton_submit">
             <property name="minimumSize">
              <size>
               <width>65</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>65</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Microsoft YaHei</family>
               <pointsize>-1</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>创建</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="IconButton" name="pushButton_cancel">
             <property name="minimumSize">
              <size>
               <width>65</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>65</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Microsoft YaHei</family>
               <pointsize>-1</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>取消</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WinButton</class>
   <extends>QPushButton</extends>
   <header>WinButton.h</header>
  </customwidget>
  <customwidget>
   <class>IconButton</class>
   <extends>QPushButton</extends>
   <header>IconButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Animism.qrc"/>
 </resources>
 <connections/>
</ui>
